.gitignore

```diff
@@ -7,3 +7,5 @@ venv/
 .pytest_cache/
 **__pycache__/
 db.sqlite3
+taxi/migrations/
```

Comment:
Consider ignoring only the migration files and not the entire `migrations` directory. Ignoring the entire directory might lead to missing essential migration files that are necessary for setting up the database schema in different environments. You can ignore specific migration files like `*.pyc` or `*.py` files that are not needed.